vignettes/articles/theme.Rmd

```diff
@@ -20,11 +26,17 @@ output:
 ---
 ```
 
-This screenshot shows the result of applying the above `theme` to [this example dashboard](https://gist.github.com/cpsievert/04bedbe9af652fab8c24a5e5eee0d721) (subsequent screenshots also use this example, but swap out different `theme`s).
+<!--Carson- can you expand on when you need to specify Bootstrap version- should we recommend always? what is the default if not stated?-->
```

Comment:
The "safe/simple" answer is always, but it's not really needed if you're only using the named arguments of `bs_theme()`  (since we're going to support these for any version)